jobspec/parse.go

```diff
@@ -619,6 +619,10 @@ func parseResources(result *structs.Resources, list *ast.ObjectList) error {
 		result.Networks = []*structs.NetworkResource{&r}
 	}
 
+	// Combine the parsed resources with a default resource block.
+	min := structs.DefaultResources()
```

Comment:
Talked about defaulting with Armon a bit and the consensus is the api endpoint should do as little defaulting as possible